src/constants/timeConstants.js

```diff
@@ -0,0 +1,17 @@
+const MILLISECONDS_PER_SECOND = 1000;
+const MILLISECONDS_PER_MINUTE = MILLISECONDS_PER_SECOND * 60;
+const MILLISECONDS_PER_HOUR = MILLISECONDS_PER_MINUTES * 60;
+const MILLISECONDS_PER_DAY = MILLISECONDS_PER_HOUR * 24;
+const MILLISECONDS_PER_WEEK = MILLISECONDS_PER_DAY * 7;
+const MILLISECONDS_PER_MOUNTH = MILLISECONDS_PER_WEEK * 30;
+const MILLISECONDS_PER_YEAR = MILLISECONDS_PER_MOUNTH * 12;
```

Comment:
Here we are carrying too much uncertainty. It's not a problem to define the month with 30 days since this is the definition of a commercial month. It's not a problem to define a year with 365 days either for the same reason.  But if we define a month with 30 days we cannot multiply it by 12 and call it a year, because it would be a year of 12 months of 30 days. It would result in a year with 360 days, which is wrong.
Also, we cannot define a month with 4 weeks. It would result in a month with 28 days and a year with 336 days.

```suggestion
const MILLISECONDS_PER_SECOND = 1000;
const MILLISECONDS_PER_MINUTE = MILLISECONDS_PER_SECOND * 60;
const MILLISECONDS_PER_HOUR = MILLISECONDS_PER_MINUTES * 60;
const MILLISECONDS_PER_DAY = MILLISECONDS_PER_HOUR * 24;
const MILLISECONDS_PER_WEEK = MILLISECONDS_PER_DAY * 7;
const MILLISECONDS_PER_MOUNTH = MILLISECONDS_PER_DAY * 30;
const MILLISECONDS_PER_YEAR = MILLISECONDS_PER_DAY * 365;
```